### PR TITLE
Change example text in create-a-form-element.md

### LINKED ISSUE #40523: Change example for Basic HTML and HTML5: Create a Form Element
Hi there! I would like to discuss a change to the example given in the create a form element challenge. This trips up a lot of people and is always brought up in the forum. 

https://forum.freecodecamp.org/search?q=create%20a%20form%20element%20after%3A2020-12-01

I think a small change to the example would help a lot of people. 

**Describe your problem and how to reproduce it:**
Existing example: `<form action="/url-where-you-want-to-submit-form-data"></form>`
Possible change: `<form action="https://submit-form-data.com"><input></form>`

Using https:// at the beginning of the action will hopefully get people to correctly use the url provided. Placing the input inside the form will hopefully fix the confusion around nesting. 

I think a lot of beginners look to the examples to help them with the challenges. So I think making these changes should help. While it might look like we are giving them the answer with this proposed change, there have been previous html challenges where the example was basically the answer. 

https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/add-images-to-your-website
https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/link-to-external-pages-with-anchor-elements
 

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/create-a-form-element

**Tell us about your browser and operating system:**
* Browser Name: 
* Browser Version: 
* Operating System: 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by thdU**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40523

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md
Comment (ShaunSHamilton) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md:14]:
````suggestion
```html
<form action="/url-where-you-want-to-submit-form-data">
  <input>
</form>
```
````

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Hey there,

Thanks, for this PR. However, I would prefer this formatting on the code. It helps seal the idea of nesting (with the indentation)

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md
Comment (thdU) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md]: Thanks, yeah that's definitely better. Just committed that change.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 . Thanks, for making that change.

For future reference, if a suggestion is made on a GitHub PR, it is easiest to click the _Commit_ button on the suggestion.

**Comment by thdU** [comment]
OK, thank you for the approval. How does the PR get merged-in from here? Github says, "Only those with write access to this repository can merge pull requests.".

**Comment by ShaunSHamilton** [comment]
Our workflow usually involves at least 2 members approving a PR, before it is merged. Also, I will need to confirm we do not have a CODE FREEZE on - if we do, no curriculum-related PRs are merged, until the freeze is over.

As it stands, there is nothing more you need to do. Thanks, again.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Thank you for your pull request! We look forward to additional contributions!

Please note that we are currently in a code freeze and are not merging PRs to the curriculum files. Once this is lifted we will be able to merge your PR.

**Comment by nhcarrigan** [comment]
Congratulations on your first pull request to this project! We look forward to future contributions!

**Pull request merged**
